### PR TITLE
Update dependencies when packaging

### PR DESCRIPTION
**Pull request by bac**
Rather than list items in setup.py, use requirements.txt and test-requirements.txt.
Explicitly reject libsass as it is only needed for building but not running the app.
Bump version to 1.9.0 in anticipation of beta 1 release.
Clean up some Makefile issues.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3646/
Test FAILed.

**Comment by makyo** [comment]
rerunning: http://ci.jujugui.org:8080/job/juju-gui/3647/

File: Makefile [hunk 1]
```diff
@@ -277,22 +278,22 @@ update-downloadcache: $(CACHE)
 # DEPS
 ######
 # Use the pyramid install dir as our indicator that dependencies are installed.
-$(PYRAMID): $(PY) $(CACHE)
+$(PYRAMID): $(PY) $(CACHE) requirements.txt
 	$(call PIP,requirements.txt)
+	@touch $(PYRAMID)
```
Comment (hatched) [inline Makefile]: Is this canary necessary? In my (limited) experience with pip it appears that it only installs items which aren't installed and skips ones which are already installed.

**Comment by hatched** [comment]
I don't think that the markupsafe package in requirements.txt is being used. It's not included anywhere...

File: Makefile (hunk 1 above)
Comment (bac) [inline Makefile]: PYRAMID is the pre-requisite for the target.  If the pre-req is not updated then that target will execute over and over.  It is not a canary.

**Comment by hatched** [comment]
Looks good! :+1:

**Comment by bac** [comment]
The failures seen in CI are transient node failures.  Let's see it the merge job works.

Thanks for the review.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1241

**Pull request merged**